localstack/services/sqs/sqs_listener.py

```diff
@@ -196,6 +196,13 @@ def get_external_port(headers, request_handler):
     return request_handler.proxy.port
 
 
+def validate_empty_message_batch(data):
+    data = to_str(data).split('Entries=')
```

Comment:
Not sure how robust this is, e.g., if the URL-encoded payload string in `data` is something like `foo=123&Entries=&bar=456`. Let's avoid parsing URL-encoded strings manually via `str.split(..)`.

We can actually use `req_data` from line 211, which already contains the parsed request data. We can call this function like this from line 285:
```
if validate_empty_message_batch(req_data):
    ...
```
... and then change this function to:
```
def validate_empty_message_batch(req_data):
    return bool(req_data.get('Entries'))
```

Reply:
_I have actually explored that option first , the Entries data is not present as part of req_data dict The params present in the req_data are depending upon the input data of the message.
the data looks something like
 'SendMessageBatch.member.id.Value' & 'SendMessageBatch.member.DataType.Value'  
where id being part of request.Is there anything which we can do ?
I think this is because the urlParsing is different for get Method and Post Method.